exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java

```diff
@@ -295,7 +295,7 @@ public FragmentExecutor getFragmentRunner(final FragmentHandle handle) {
    */
   private class StatusThread extends Thread {
     public StatusThread() {
-      setDaemon(true);
+      setDaemon(false);
```

Comment:
Since the StatusThread is started by "main" thread by default the StatusThread will be non-Daemon, so we can just remove the setDaemon call.